### PR TITLE
Grid component added

### PR DESCRIPTION
**Pull request by Zissan**
**Basic view:**
Screenshot 2023-01-23 at 6 12 10 PM

**API:**
Screenshot 2023-01-23 at 6 12 21 PM

### CODE REVIEW

File: src/components/Grid/Grid.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@import "./../../base/base.scss";
+
+.nitrozen-grid-container {
+  display: grid;
+  //   dynamic class for props template
```
Comment (Zissan) [inline src/components/Grid/Grid.scss:5]: Please remove the commented code.

Comment (Zissan) [reply src/components/Grid/Grid.scss:5]: Use rem in place of px

File: src/components/Grid/Grid.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      setResponsiveTemplate(tabletTemplate);
+    }
+
+    if (getWindowWidth() <= 480) {
+      setResponsiveTemplate(mobileTemplate);
+    }
+    // console.log("getWindowWidth()", getWindowWidth());
+  }, []);
+
+  const getWindowWidth = () => {
```
Comment (Zissan) [inline src/components/Grid/Grid.tsx:48]: Move this function as this will introduce performance issue.

File: src/components/Grid/Grid.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    }
+
+    if (getWindowWidth() <= 768) {
+      setResponsiveTemplate(tabletTemplate);
+    }
+
+    if (getWindowWidth() <= 480) {
+      setResponsiveTemplate(mobileTemplate);
+    }
+    // console.log("getWindowWidth()", getWindowWidth());
```
Comment (Zissan) [inline src/components/Grid/Grid.tsx:45]: `
let template = desktopTemplate;
const windowWidth = getWindowWidth()
if(windowWidth <=  768) { ... } else if () {...}

setResponsiveTemplate(template)
`

File: src/components/Grid/Grid.tsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        justifyItems: justify,
+        ...(style ?? {}),
+      }}
+      className={`nitrozen-grid-container nitrozen-grid-font-size ${
+        className ?? ""
+      }`}
+      {...restProps}
+    >
+      {children}
+      {/* <div className="item1">
```
Comment (Zissan) [inline src/components/Grid/Grid.tsx]: Please remove

**Comment by ShahRohan27** [comment]
Closing this PR due to inactivity @Zissan

**Pull request closed without merging**